pyproject.toml

```diff
@@ -51,6 +52,7 @@ tomli = "^2.0.1"
 dateparser = "^1.1.8"
 urllib3 = "1.26.15"
 importlib-resources = "^5.12.0"
+beautifulsoup4 = "^4.11.1"
```

Comment:
bs4 seems to be only used in testcases, so it should be a dev-dependency, right?

In general, I would vote for keeping the dependencies limited, but that's up to @tngraf to decide. :-)

Reply:
bs4 is used here.
https://github.com/sachinshaji/capycli-granularity-download/blob/feat/findsources/capycli/bom/findsources.py#L196